src/main/java/gel/Storage.java

```diff
@@ -0,0 +1,119 @@
+package gel;
+
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+
+import gel.task.Deadline;
+import gel.task.Event;
+import gel.task.Task;
+import gel.task.Todo;
+
+/**
+ * Storage deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+
+    private String filePath;
+    private String directoryPath;
+
+    /**
+     * Constructor of Storage class.
```

Comment:
I'm not too sure if this rule applies to constructors as well, but should the first sentence of the method header comment start in the form of "Constructs ..."? See [Java Coding Standard -> Comments](https://se-education.org/guides/conventions/java/index.html#comments).